client/cody/src/chat/protocol.ts

```diff
@@ -56,6 +57,7 @@ export interface AuthStatus {
     authenticated: boolean
     hasVerifiedEmail: boolean
     requiresVerifiedEmail: boolean
+    isNetworkError?: boolean
```

Comment:
```suggestion
    showNetworkError?: boolean
```
Nit: match with the show error above 😃